lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -540,58 +540,6 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             , expectField #withdrawals (`shouldSatisfy` (withdrawalWith External))
             ]
 
-    it "TRANS_NEW_CREATE_03b - Withdrawal from external wallet" $ \ctx -> runResourceT $ do
```

Comment:
At the end of this pull request, there seems to be a mismatch between implementation and documentation: 🤔 The swagger excludes the external withdrawals from the endpoint, but the `ApiConstructTransactionData` type still uses `ApiWithdrawalPostData`, which means that it will happily _parse_ withdrawals to an external wallet.

Maybe going all the way of removing support is out of scope for this pull request, but I recommend doing it in a follow-up pull request, then.

Reply:
I introduced `ApiSelfWithdrawalPostData` to address this in the current PR and adjusted all code and testing